target/linux/ath79/dts/ar9331_hak5_wifi-pineapple-nano.dts

```diff
@@ -24,11 +29,6 @@
 			gpio-export,name = "microsd-detect";
 			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
 		};
-
-		usb-alarm {
-			gpio-export,name = "usb-alarm";
-			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
```

Comment:
It seems that it output low level here. I guess the beeper is triggered by high level, so it should be converted to
```
	beeper {
		compatible = "gpio-beeper";
		gpios = <&gpio 20 GPIO_ACTIVE_HIGH>;
	};
```
to keep the beeper off by default? 

Request for comment.